streaming/python/tests/test_hybrid_stream.py

```diff
@@ -31,12 +31,9 @@ def test_hybrid_stream():
         "../../../bazel-bin/streaming/java/all_streaming_tests_deploy.jar")
     jar_path = os.path.abspath(jar_path)
     print("jar_path", jar_path)
-    java_worker_options = json.dumps(["-classpath", jar_path])
-    print("java_worker_options", java_worker_options)
     assert not ray.is_initialized()
     ray.init(
-        job_config=ray.job_config.JobConfig(code_search_path=sys.path),
-        _java_worker_options=java_worker_options)
+        job_config=ray.job_config.JobConfig(code_search_path=[jar_path]))
```

Comment:
Please update other streaming test cases with `code_search_path` configured as well. I believe we chan change them from `ray.init(job_config=ray.job_config.JobConfig(code_search_path=sys.path))` to `ray.init()`.